.githooks/pre-commit

```diff
@@ -0,0 +1 @@
+cargo fmt && cargo fix --allow-dirty --allow-staged
```

Comment:
cargo fmt ok, cargo fix takes too long to be run on each commit

Reply:
Agree. Removed.